cmd/nv2/gencert.go

```diff
@@ -0,0 +1,209 @@
+package main
+
+import (
+	"crypto"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"log"
+	"math/big"
+	"net"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	// This needs to be configurable. Once the location of the
+	// configuration is finalized this parameter and file locations
+	// should be exposed as options on the cli with mock tests.
+	KEYS_BASE_DIR = ".notation"
+)
+
+var certificatesCommand = &cli.Command{
+	Name:  "certificates",
+	Usage: "Commands to manage certificates",
+	Subcommands: []*cli.Command{
+		generateCertCommand,
+	},
+}
+
+var generateCertCommand = &cli.Command{
+	Name:      "generate",
+	Usage:     "Generates a test crt and key file.",
+	ArgsUsage: "[host]",
+	Flags: []cli.Flag{
+		&cli.IntFlag{
+			Name:     "rsaBits",
+			Usage:    "--rsaBits 3072",
+			Required: false,
+			Value:    3072,
+		},
+		&cli.StringFlag{
+			Name:     "not-after",
```

Comment:
Will it be a better customer experience to take validity period `validForDays` instead of actual date?
Pros: 
* easy to associate default value of 1 year
* No need to worry about not-after being in past.

```
certificates generate --not-after 2006-01-02T15:04:05-07:00
certificates generate --valid-for-days 365
```

Reply:
`validForDays` might not be a good one. For testing purposes, people may want short-lived certs like serveral hours or minutes.